compiler/modules/front.syntax/ast.bal

```diff
@@ -202,6 +202,13 @@ public type CheckingExpr record {|
     Expr operand;
 |};
 
+public type CheckingStmt record {|
+    // JBUG can't include CheckingExpr
+    // *CheckingExpr;
+    CheckingKeyword checkingKeyword;
+    FunctionCallExpr|MethodCallExpr|CheckingExpr operand;
```

Comment:
Shouldn't the type be `CallStmt operand;`? This allows a CheckingExpr that contains anything.

Reply:
Yes, that is more in line with the spec. I did it this way thinking then it wouldn't be a subtype of `CheckingExpr` because jBallerina gave an error, I guess due to structural typing it has to be a subtype.

Let me change it.